get_shanbay.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import requests
+import pendulum
+
+from github import Github
+
+# the shanbay issue index is 12
+GET_UP_ISSUE_NUMBER = 12
+# update your own shanbay user_name
+SHANBAY_USERNAME = "zlurjj"
```

Comment:
empty string here, i will change it to mine

Reply:
already setting to empty , please check out it ~